vsc-extension-quickstart.md

```diff
@@ -2,22 +2,23 @@
 
 ## What's in the folder
 
-* This folder contains all of the files necessary for your extension.
-* `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
-* `snippets/snippets.code-snippets` - the file containing all snippets.
+- This folder contains all of the files necessary for your extension.
+- `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
+- `snippets/snippets.code-snippets` - the file containing all snippets.
+- `snippets/snippetsesm.code-snippets` - the file containing all ESM snippets.
```

Comment:
I don't really care about this md file, it came with the generator, I'll just delete it.